logger/logger.go

```diff
@@ -0,0 +1,42 @@
+package logger
+
+import "fmt"
+
+var log *Logger
+
+// Logger provided simple logger.
+type Logger struct {
+	verbose bool
+}
+
+// Setup configures logger
+func Setup(verbose bool) *Logger {
+	log = &Logger{
+		verbose,
+	}
```

Comment:
Let's make this initialization thread-safe just in case we decide to introduce concurrency in the tool in the future.

This can be achieved using a `sync.Mutex` similar to [this gist](https://gist.github.com/jeffotoni/8bc07d67519751296cee98c4ff1c311a#file-golang-singleton-goroutine-new-2-go)

No need for the singleton pattern, just the explicit locking.

For more info, check out [this article](https://medium.com/golang-issue/how-singleton-pattern-works-with-golang-2fdd61cd5a7f)

Reply:
Thank you for pointing to new problems and patterns. I am newbie in golang and all new knowledge is important for me.